docs/estela/api/estela-endpoints.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: page
+title: Enpoints
```

Comment:
Fix this typo (`Endpoints` instead of `Enpoints`) and this is ready to merge.

Reply:
Done